### PR TITLE
Python 3: remove autotest dependency and do some porting for libguestfs

### PR DESCRIPTION
**Pull request by czhang0**
1) Remove autotest dependency
2) Use 2to3 to do some port

Signed-off-by: cuzhang <cuzhang@redhat.com>

### CODE REVIEW

**Comment by czhang0** [comment]
I  will commit new patch to replace all subprocess.* in the code.

File: libguestfs/tests/guestfish_misc.py [hunk 1]
```diff
@@ -175,14 +175,14 @@ def test_download(vm, params):
     gf.download("/src.txt", "%s" % dest)
     gf.close_session()
 
-    content = commands.getoutput("cat %s" % dest)
-    commands.getstatus("rm %s" % dest)
+    content = process.getoutput("cat %s" % dest)
+    process.system("rm %s" % dest)
```
Comment (chunfuwen) [inline libguestfs/tests/guestfish_misc.py]: maybe change to process.getstatus()

File: libguestfs/tests/virt_file_operations.py [hunk 2]
```diff
@@ -31,14 +30,14 @@ def test_virt_tar_in(vm, params):
     # Create a file on host
     try:
         open(path, 'w').write(content)
```
Comment (chunfuwen) [inline libguestfs/tests/virt_file_operations.py]: change to with open

File: libguestfs/tests/virt_file_operations.py [hunk 3]
```diff
@@ -165,8 +165,8 @@ def test_virt_copy_in(vm, params):
...
```
Comment (chunfuwen) [inline libguestfs/tests/virt_file_operations.py]: with open

File: libguestfs/tests/guestfish_misc.py (hunk 1 above)
Comment (czhang0) [reply libguestfs/tests/guestfish_misc.py]: Thanks for your comments.
But there is no process.getstatus() is implemented. I will add shell=True parameter for process.system() to enable using shell to run command.

File: libguestfs/tests/guestfish_misc.py [hunk 4]
```diff
@@ -210,14 +210,14 @@ def test_download_offset(vm, params):
     gf.download_offset("/src.txt", "%s" % dest, 0, len(string))
     gf.close_session()
 
-    content = commands.getoutput("cat %s" % dest)
-    commands.getstatus("rm %s" % dest)
+    content = process.getoutput("cat %s" % dest)
+    process.system("rm %s" % dest)
```
[the remark this answers is not in the document]
Comment (czhang0) [reply libguestfs/tests/guestfish_misc.py]: Same above. Thanks.

File: libguestfs/tests/virt_file_operations.py (hunk 3 above)
Comment (czhang0) [reply libguestfs/tests/virt_file_operations.py]: Ok, I will do it.

File: libguestfs/tests/virt_file_operations.py (hunk 2 above)
Comment (czhang0) [reply libguestfs/tests/virt_file_operations.py]: Ok, I will do it.

**Pull request merged**